modules/twinklespeedy.js

```diff
@@ -153,6 +163,33 @@ Twinkle.speedy.initDialog = function twinklespeedyInitDialog(callbackfunc) {
 					}
 				]
 			} );
+		form.append( {
+			type: 'checkbox',
+			list: [
+				{
+					label: 'Delete under multiple criteria',
+					value: 'delmultiple',
+					name: 'delmultiple',
+					tooltip: "When selected, you can select several criteria that apply to the page. For example, G11 and A7 are a common combination for articles.",
+					event: function( event ) {
+						Twinkle.speedy.callback.modeChanged( event.target.form );
+						event.stopPropagation();
+					}
+				}
+			]
+		} );
+		form.append( {
+				type: 'checkbox',
+				list: [
+					{
+						label: 'Open user talk page on submit',
+						value: 'openusertalk',
+						name: 'openusertalk',
+						tooltip: 'Use this is override your preferences for the selected rationale',
```

Comment:
You could loop through the selected rationale and see if your preferences are all the same for those criteria, but I don't really think that is that advantageous. In the case of multiple we'll just let the admin decide what to do. Maybe it should default to checked, though